### ISSUE TITLE
Looking for help to develope a Websocekt multichannel Oscilloscope

### ISSUE THREAD

**Issue by williamesp2015** [cnlohr/colorchord#6]
Your project is awesome and I would be grateful if you kindly help to develop a websocket server with high speed (150 Hz) multichannel (up to 10 channels)  oscilloscope.

**Comment by cnlohr** [comment]
150Hz sampling?  Or higher frequency sampling, update display at 150Hz?  How would data get into the ESP?

**Comment by williamesp2015** [comment]
Data will send by ESP (Websocket client) to a Websocket Server (web-base not ESP) and display by highest frequency on oscilloscope (can display by lower frequency let say 30 Hz) and save into database by highest frequency (150 times/sec).

**Comment by cnlohr** [comment]
Hey, sorry, I've been so busy for so long.  Depending on your input rate, and if you need multiple channels you may need to look into using I2S to capture.  I went down that road briefly, but didn't dig into it too much, so I wouldn't be of too much help.  I am hoping to revisit it but don't hold out much hope.

**Comment by neobrien** [comment]
yeah making an oscope i think is a good industrial use of the ESP
